### ISSUE TITLE
Escaped quotes in Ergo string are output with a spurious \

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/ergo#665]
**Describe the bug**
When returning strings containing (escaped) quotes, they are returned with a spurious `\`.

**To Reproduce**
In `ergotop` try: `return "This is a \" quote"`.

**Expected behavior**
Should return the same string as the input.

**Screenshots**

<img width="842" alt="Screenshot 2019-10-07 at 8 46 29 AM" src="https://user-images.githubusercontent.com/670099/66312886-687ec400-e8df-11e9-86d1-8ccd1fd3c685.png">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser N/A
 - Version 0.20.*

**Comment by Shivansh2407** [comment]
please assign this to me

**Comment by Shivansh2407** [comment]
Thanks..Can you please specify which file needs this fix?

**Comment by jeromesimeon** [comment]
@Shivansh2407 It certainly requires some investigation. I'm not even clear whether this is a compile-time or execution-time related problem. It might be worth trying the same thing with the JavaScript backend for comparison, since this could be a `ergotop` only issue.

**Comment by jeromesimeon** [comment]
Some additional data point. The escaping seems to work fine on the JavaScript backend:
```
bash-3.2$ cat examples/helloworld/logic/logic.ergo 
namespace org.accordproject.helloworld

contract HelloWorld over TemplateModel {
  // Simple Clause
  clause helloworld(request : Request) : Response {
    return Response{ output: "Hello \"" ++ contract.name ++ "\" (" ++ request.input ++ ")" }
  }
}
bash-3.2$ ./packages/ergo-cli/index.js execute --template examples/helloworld --data examples/helloworld/data.json --request examples/helloworld/request.json 
{"clause":"orgXaccordprojectXhelloworldXHelloWorld","request":{"$class":"org.accordproject.helloworld.Request","input":"Accord Project"},"response":{"$class":"org.accordproject.helloworld.Response","output":"Hello \"Fred Blogs\" (Accord Project)","transactionId":"58f5f261-454e-40d2-8b50-dc40d8aa0641","timestamp":"2019-10-14T18:18:16.564Z"},"state":{"$class":"org.accordproject.cicero.contract.AccordContractState","stateId":"org.accordproject.cicero.contract.AccordContractState#1"},"emit":[]}
```

**Comment by jeromesimeon** [comment]
Fixed in the `0.20` branch https://github.com/accordproject/ergo/commit/39cc3988bfe27a12675bebfa538c8908e81ad779

**Comment by jeromesimeon** [comment]
<img width="682" alt="Screenshot 2019-10-14 at 3 16 38 PM" src="https://user-images.githubusercontent.com/670099/66776947-a2c70300-ee95-11e9-8e82-be5463c778b2.png">
